Mathlib/SetTheory/ZFC/Ordinal.lean

```diff
@@ -89,4 +97,42 @@ theorem isTransitive_iff_subset_powerset : x.IsTransitive ↔ x ⊆ powerset x :
 
 alias ⟨IsTransitive.subset_powerset, _⟩ := isTransitive_iff_subset_powerset
 
+/-- A set `x` is a von Neumann ordinal when it's a transitive set, that's transitive under `∈`. We
+will prove that this further implies that `x` is well-ordered under `∈`.
+
+The transitivity condition is written in an even weaker form, where `a ∈ b` and `b ∈ c` imply
+`a ∈ c` when only `c ∈ x` and not `a ∈ x` or `b ∈ x` are known a priori. -/
```

Comment:
Doesn't `x.IsTransitive` automatically give that `b ∈ c` together with `c ∈ x` imply `b ∈ x`? If so, I'm not sure I'd phrase this as "written in an even weaker form". Perhaps "The transitivity condition can be written without assuming `a ∈ x` and `b ∈ x`, and `isOrdinal_iff_isTrans` shows it is equivalent to the usual one." What do you think?

Reply:
I like that phrasing better.